.github/workflows/ci.yaml

```diff
@@ -32,6 +32,9 @@ jobs:
       - name: Lint
         run: pnpm lint
 
+      - name: Install Playwright Dependencies
+        run: pnpm exec playwright install chromium --with-deps
```

Comment:
I think it's a bit overkill to run playwright tests on every platform times node version. 

How about we create a separate e2e section that only runs on `ubuntu-latest` and only run the `e2e` tests?

Reply:
Yup, I'll change this. Though I wonder if we will miss Windows related bugs like below in future:
- https://github.com/stackblitz/tutorialkit/issues/209
- https://github.com/stackblitz/tutorialkit/issues/183
- https://github.com/stackblitz/tutorialkit/issues/182

Reply:
I think those issues could be captured differently. Like with an integration test that run a few simple sanity checks against the generated CSS. (like checking that a class is present for instance)

As long as the same JS / CSS and html are identical between platform, we can afford to only run UI tests on `ubuntu-latest`.

Reply:
> As long as the same JS / CSS and html are identical between platform

And the generated JSON's like in #209.

Reply:
Oh yeah :100: